snap/snapcraft.yaml

```diff
@@ -65,19 +73,18 @@ parts:
       snapcraftctl build
     plugin: cmake
     configflags: ["-DRUN_IN_PLACE=FALSE", "-DCMAKE_BUILD_TYPE=Release"]
+    after:
+      - irrlicht
     build-packages:
       - cmake
       - gcc
       - g++
       - gettext
-      - imagemagick
-      - libbz2-dev
       - libcurl4-gnutls-dev
       - libfreetype6-dev
       - libglu1-mesa-dev
       - libirrlicht-dev
```

Comment:
This needs to be dropped if we're to use the fork from minetest.
```suggestion
```

Reply:
I forgot to remove that, fixed.